main.nf

```diff
@@ -42,11 +42,31 @@ workflow NFCORE_PHASEIMPUTE {
     ch_versions // channel: versions of software used
 
     main:
+
+    //
+    // Initialise input channels
+    //
+
+    input_impute         = Channel.empty()
+    input_simulate       = Channel.empty()
+    input_validate       = Channel.empty()
+
+    if (params.step == "impute") {
+        input_impute   = ch_input
+    } else if (params.step == "simulate" || params.step == "all") {
+        input_simulate = ch_input
+    } else if (params.step == "validate") {
+        input_validate = ch_input
```

Comment:
I understand this as a way to always have `--input samplesheet.csv` but based on the step, it will require a different type of samplesheet. Impute and simulate require `.bam` as inputs, while validate requires 2 `.vcf`. 

To be run as a standalone process `--step validate ` would require 2 input vcf (imputed and truth) in `--input samplesheet.csv`. However, in its current version `schema_input.json` cannot handle that case. How could the schema look like? something like `sample, vcf_truth, vcf_imputed`?

Reply:
For imputation it should also work with vcf.
The input should effectively work with those extension.
I'll update it.